### ISSUE TITLE
Instructions to get started in README

### ISSUE THREAD

**Issue by rolfkleef** [xspec/xspec.github.io#7]
I've added [a pull request](https://github.com/xspec/xspec.github.io/pull/5) with two things:

- Some README instructions to help people get started with this repo.
- .gitignore for locally generated files (when testing) that shouldn't go into git.

It might be good to make `hugo` the default branch in the settings of the repository:

- It will show the source rather than the generated site (and the README on that branch).
- If someone creates a pull request, I think it will pick the hugo branch by default.

And perhaps as a by the way @cirulls: I had [another pull request](https://github.com/xspec/xspec.github.io/pull/6) for the CNAME file, but located in the `'hugo` branch under `static`. I just closed that, but now am thinking that when the Github Action regenerates the static site, your commit of that file on the `master` branch will be overwritten. So perhaps we need that file on the `hugo` branch after all.

**Comment by cirulls** [comment]
Yes, the CNAME file must be committed to the Hugo branch as well, thanks for creating the PR, I'll review it in the next days but it looks fine. It was created automatically in the `master` branch when I did the custom DNS configuration. I agree that the `hugo` branch should be the default one as PRs are going to be created against that branch. I'll change these settings by the end of this week.

**Comment by cirulls** [comment]
Just to keep track of work done elsewhere: for README and `.gitignore` see #5, for CNAME and branche see https://github.com/xspec/xspec.github.io/issues/4#issuecomment-654069528.
